raspi3.yaml

```diff
@@ -66,6 +66,7 @@ steps:
   - apt: install
     packages:
     - ssh
+    - parted
```

Comment:
This looks like a left-over which could be removed?

Reply:
Nope.
partprobe is part of parted.